context-free.md

```diff
@@ -2,21 +2,55 @@
 
 ##  Who is this programming language for?
 
+Context Free is useful for people who are interested in creating
+computer-generated art.  This could have uses for graphical designs or just
+expressing creativity.  Other features in Context Free could interest animators
```

Comment:
Interesting! I hadn't thought of animators at all. I wish you had elaborated on this point, I don't immediately see how Context Free would help animators. I might be missing a helpful feature of Context Free, but I think the animator would have to render many many images with slight modifications to the options of each shape, which would be a tedious process. If I absolutely wanted to animate something with Context Free, I'd use some other language to generate Context Free code.